hbase/src/main/java/com/yahoo/ycsb/db/HBaseClient10.java

```diff
@@ -133,6 +135,20 @@ public void init() throws DBException
             throw new DBException("No columnfamily specified");
         }
         _columnFamilyBytes = Bytes.toBytes(_columnFamily);
+
+      // Terminate right now if table does not exist, since the client
+      // will not propagate this error upstream once the workload
+      // starts.
+      String table = com.yahoo.ycsb.workloads.CoreWorkload.table;
+      try
+	  {
+	      HTable ht = new HTable(config, table);
+	      HTableDescriptor dsc = ht.getTableDescriptor();
```

Comment:
We might want to use [`Connection.getTable(...).getTableDescriptor()`](https://hbase.apache.org/apidocs/org/apache/hadoop/hbase/client/Table.html#getTableDescriptor()) for the 1.0 client.